TOC/Proposal/voting-rules.md

```diff
@@ -0,0 +1,23 @@
+# Voting Rules for TOC Members
+
+This guideline is used for the decision-making process of TOC members in the TiDB community. Refer to Apache's voting rules, decisions are usually made by [lazy consensus](https://www.apache.org/foundation/glossary.html#LazyConsensus). If no consensus can be reached, the matter may be resolved by a majority vote. And potentially highly-controversial matters are recommended to be resolved by a majority vote, too.
+
+## Voting
+
+For all votes, voting must be open for at least 2 days. The deadline should be clearly stated in the call to vote. Votes are clearly indicated by a pull request adding an entry under the proposal folder.
+
+All TOC members have the right to make proposals. Only official members have the right to vote, and each has 1 vote. For public discussions, non-binding votes from the community are also highly appreciated.
+
+### Lazy Consensus
+
+The default decision-making mechanism in TiDB Community is [lazy consensus](https://www.apache.org/foundation/glossary.html#LazyConsensus). This means that any proposal is considered supported by the community as long as nobody objects.
```

Comment:
You can write

```
The default decision-making mechanism in TiDB Community is lazy consensus, inspired by [Apache LazyConsensus](https://www.apache.org/foundation/glossary.html#LazyConsensus).
```

... so we clarify it is an inheritance, not the same.

Reply:
OK, thanks.